artifacts.md

```diff
@@ -6,7 +6,19 @@ description: Encapsulating data in Aqueduct
 
 _Artifacts_ are thin wrappers around data objects in Aqueduct. Wrapping data in an Artifact allows Aqueduct to track that artifact across workflow runs help you understand how your data is changing over time.&#x20;
 
-Aqueduct currently supports `TableArtifacts`, which are encapsulations of tabular or relational data, and [`ParameterArtifacts`](workflows/parameterizing-a-workflow.md), which wrap mutable json values. We plan to add support for other data types in the future.
+All _Artifacts_ that are not of Type `GenericArtifact` are already aware of the underlying data types they hold. For example, the `BooleanArtifact` holds a `boolean`. A `GenericArtifact` holds a generic data type that is not encapsulated by the other explicit artifacts. A full list of Artifact data types within the system can be found below:
```

Comment:
hmm I think it flows much better if we don't talk about `GenericArtifact` and that stuff at all. To me, it reads like implementation detail that they should't care about. We should just have "Here is the full list of supported Artifact data types:" and then the list below.

And I think for the boolean and numeric entries, you can add a note on the side like
"boolean: This is also the type returned by Check operators.", etc